src/features/pdf/AutomaticPDFLayout.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+
+import { SummaryComponent } from 'src/components/summary/SummaryComponent';
+import css from 'src/features/pdf/PDFView.module.css';
+import { GenericComponent } from 'src/layout/GenericComponent';
+import { topLevelComponents } from 'src/utils/formLayout';
+import type { ILayoutCompInstanceInformation } from 'src/layout/InstanceInformation/types';
+import type { ILayoutComponent, ILayoutComponentOrGroup, ILayouts } from 'src/layout/layout';
+
+const summaryComponents = new Set([
+  'AddressComponent',
+  'AttachmentList',
+  'Checkboxes',
+  'Custom',
+  'DatePicker',
+  'Dropdown',
+  'FileUpload',
+  'FileUploadWithTag',
+  'Group',
+  'Input',
+  'List',
+  'Map',
+  'MultipleSelect',
+  'RadioButtons',
+  'TextArea',
+]);
+
+const presentationComponents = new Set(['Header', 'Paragraph', 'Image', 'Panel']);
+
+const renderComponents = new Set([...summaryComponents, ...presentationComponents]);
```

Comment:
Yikes! :laughing: Prime candidates for the `LayoutComponent` class. :partying_face:

Reply:
Haha, I know 😆